src/main/scala/io/iohk/ethereum/vm/VM.scala

```diff
@@ -13,9 +14,9 @@ object VM {
     * @param context context to be executed
     * @return result of the execution
     */
-  def run(context: ProgramContext): ProgramResult = {
-    val finalState = run(ProgramState(context))
-    ProgramResult(finalState.returnData, finalState.storage, finalState.error)
+  def run(env: ExecEnv, storage: Storage): ProgramResult = {
+    val finalState = run(ProgramState(env, storage))
+    ProgramResult(finalState.returnData, finalState.storage, Seq(), Seq(), finalState.error)
```

Comment:
shouldn't it be `finalState.internalTransactions` instead of empty `Seq()` (same for acc to delete)